task/06-conditions-n-loops-tasks.js

```diff
@@ -82,10 +93,11 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+    if (a**2 + b**2 === c**2 || a**2 + c**2 === b**2 || b**2 + c**2 === a**2 || (a === b && b === c)){
+        return true;
+    } return false;
```

Comment:
We can just `return (a**2 + b**2 === c**2 || a**2 + c**2 === b**2 || b**2 + c**2 === a**2 || (a === b && b === c))`

Please, see https://javascript.info/logical-operators